modules/my_health/app/controllers/my_health/v2/immunizations_controller.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/client'
+require 'lighthouse/veterans_health/models/immunization'
+require 'lighthouse/veterans_health/serializers/immunization_serializer'
+require 'common/client/errors'
+require 'common/exceptions'
+
+module MyHealth
+  module V2
+    class ImmunizationsController < ApplicationController
+      service_tag 'mhv-medical-records'
+
+      STATSD_KEY_PREFIX = 'api.my_health.immunizations'
+      FEATURE_TOGGLE = 'mhv_medical_records_immunizations_v2_enabled'
+
+      before_action :check_feature_toggle
+
+      def index
+        start_date = params[:start_date]
+        end_date = params[:end_date]
+
+        begin
+          response = client.get_immunizations(start_date:, end_date:)
+          # log the request for debugging
+          immunizations = Lighthouse::VeteransHealth::Serializers::ImmunizationSerializer
+                          .from_fhir_bundle(response.body)
```

Comment:
do we want to take this out before merge?

Reply:
Did log it, just  a left over comment